media-plugins/vdr-osdteletext/vdr-osdteletext-0.9.7.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit user vdr-plugin-2
+
+VERSION="2161" # every bump, new version
+
+DESCRIPTION="VDR Plugin: Osd-Teletext displays the teletext on the OSD"
+HOMEPAGE="https://projects.vdr-developer.org/projects/plg-osdteletext"
+SRC_URI="mirror://vdr-developerorg/${VERSION}/${P}.tgz"
```

Comment:
How's the state of these mirrors? 
https://projects.vdr-developer.org/attachments/download

Maybe you should just point to exact URL?

Reply:
replaced with `SRC_URI="https://projects.vdr-developer.org/attachments/download/${VERSION}/${P}.tgz"`